package/MDAnalysis/coordinates/DCD.py

```diff
@@ -522,26 +523,27 @@ def timeseries(self, asel, start=0, stop=-1, skip=1, format='afc'):
         # Check if the atom numbers can be grouped for efficiency, then we can read partial buffers
         # from trajectory file instead of an entire timestep
         # XXX needs to be implemented
-        return self._read_timeseries(atom_numbers, start, stop, skip, format)
+        return self._read_timeseries(atom_numbers, start, stop, step, format)
 
-    def correl(self, timeseries, start=0, stop=-1, skip=1):
+    def correl(self, timeseries, start=None, stop=None, step=None):
```

Comment:
see above – changing kwargs are api changes that break code – don't do that willy-nilly.